Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,296 @@
+- OpenVR
+
+The following software is required.
+
+- Microsoft Visual Studio (2017 or 2019) Community Edition or higher
+- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
+- Unity 2018.4, 2019.1 or 2019.2
+
+### Upgrading projects to 2.0.0
+
+Since the RC2 release, there have been several changes that may impact application projects,
+including some files moving to new folder locations. Breaking change details, including mitigation guidance, can be found in the [**Updating from RC2**](Updating.md) article.
+
+For the smoothest upgrade path, please use the following steps.
+
+1. Close Unity
+1. Delete **MixedRealityToolkit** (the project may not have all listed folders)
+    - MixedRealityToolkit
+    - MixedRealityToolkit.Examples
+    - MixedRealityToolkit.Extensions
+    - MixedRealityToolkit.Providers
+    - MixedRealityToolkit.SDK
+    - MixedRealityToolkit.Services
+    - MixedRealityToolkit.Tools
+    > [!Important]
+    > Do NOT delete the **MixedRealityToolkit.Generated** folder.
+1. Delete the **Library** folder
+1. Re-open the project in Unity
+1. Import the new unity packages
+    - Foundation - _Import this package first_
+    - (Optional) Tools
+    - (Optional) Extensions
+    - (Optional) Examples
+1. For each scene in the project
+    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy
+    - Select **MixedRealityToolkit -> Add to Scene and Configure**
+
+> [!Important]
+> Some profiles have been changed (properties have been added) in this release. If the project has custom
+profiles, please open them to verify that all of the updated properties are correctly configured.
+
+### What's new in 2.0.0
+
+**BoundingBox**
+
+- Improvements
+    - Normalized the handle asset size and scaling logic
+    - The handle asset is now sized to 1-meter
+    - Default values and examples are updated
+- New features
+    - Animated handle by proximity
+    - Match the HoloLens 2 shell behavior
+    - Handles can be configured to appear only when a hand is close to them
+- New example scene
+    - The BoundingBoxExample scene, in the Examples package (Demos\UX\BoundingBox\Scenes\BoundingBoxExamples.unity), shows various types of configurations
+
+> [!Important] 
+> Normalized asset handle size and scaling logic require updated handle scale values. With this change, the existing bounding box
+handle will be displayed in a very large size. Please refer to the **Bounding box handle styles** section in the [Bounding Box](README_BoundingBox.md)
+documentation for updated recommended values for the handle size. Handle configuration examples can be found in the **BoundingBoxExamples** scene.
+
+<img src="../Documentation/Images/BoundingBox/MRTK_BoundingBox_HandleStyles2.png" width="450">
+
+**Clipping Example**
+
+The ClippingExamples scene, in the Examples package (Demos\StandardShader\Scenes\ClippingExamples.unity), demonstrates using the MRTK
```

Comment:
>package [](start = 44, length = 7)

Per @thalbern comment, was the request to add the path, or a link? I think this just adds the path, and maybe that's enough?

Reply:
most of the references to files in this doc are paths. they match what is imported from the packages. links 1) dont exist yet and 2) may change as new releases are merged into mrtk_release

thoughts?